.github/workflows/kfo.yaml

```diff
@@ -0,0 +1,17 @@
+name: kube-fluentd-operator update dependencies
+on:
+  schedule:
+  - cron: '0 */1 * * *'
```

Comment:
@cw-sakamoto 
Be a random minute different from the other cron. This is because GitHub Actions has a limit of up to 20 parallel executions.

Reply:
@cw-ozaki I see, I fixed in https://github.com/chatwork/dockerfiles/pull/132/commits/a6fc6cbd7bda05923e61be9651eb0d43d78cf81e